prow/plugins/trigger/pr.go

```diff
@@ -82,6 +87,9 @@ I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that
 
 	owner := pr.Base.Repo.Owner.Login
 	name := pr.Base.Repo.Name
+	if err := ghc.AddLabel(owner, name, pr.Number, needsOkToTest); err != nil {
```

Comment:
This will create it I believe and in this case, I'm okay with that (though the colors won't match).  With the other proposal for standardizing labels across all K8s repos, this shouldn't be an issue.